cvxpy/tests/test_python_backends.py

```diff
@@ -1312,14 +1392,171 @@ def test_mul_elementwise_parametrized(self, param_backend):
         # Note: view is edited in-place:
         assert out_view.get_tensor_representation(0) == view.get_tensor_representation(0)
 
+    def test_parametrized_div(self, param_backend):
+        """
+        Continuing from the non-parametrized example when the rhs is a parameter,
+        instead of dividing known values, the matrix is split up into four slices,
+        each representing an element of the parameter, i.e. instead of
+         x11 x21 x12 x22
+        [[1   0   0   0],
+         [0   1/3 0   0],
+         [0   0   1/2 0],
+         [0   0   0   1/4]]
+
+         we obtain the list of length four, where we have the quotients at the same entries
+         but they each map to a different parameter slice:
```

Comment:
does this explanation make sense? I was basing off of mul, but they are quite different.